core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -2237,13 +2237,21 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.NUMERIC_OPTIONAL_NUMERIC,
           SqlFunctionCategory.NUMERIC);
 
-  /** The "LOG(numeric, numeric1)" function. Returns the base numeric1 logarithm of numeric. */
+  /** The "LOG(numeric[,numeric1])" function. Returns the base numeric1 logarithm of numeric. */
   @LibraryOperator(libraries = {MYSQL, SPARK})
   public static final SqlFunction LOG_MYSQL =
       SqlBasicFunction.create(SqlKind.LOG,
           ReturnTypes.DOUBLE_FORCE_NULLABLE,
           OperandTypes.NUMERIC_OPTIONAL_NUMERIC);
 
+  /** The "LOG(numeric[,numeric1])" function. Returns the base numeric1 logarithm of numeric. */
```

Comment:
The description is a bit confusing.
'Returns the logarithm of numeric1 using numeric as the base'?

Reply:
fix

Reply:
I modified it according to the official MySQL